doc/source/Tutorials/BeginnerTutorial/screengrab.py

```diff
@@ -48,9 +48,8 @@ def __delay( delay ) :
 
 # GafferBot torso in Hierarchy View
 GafferSceneUI.ContextAlgo.setExpandedPaths( script.context(), IECore.PathMatcher( [ "/GAFFERBOT", "/GAFFERBOT/C_torso_GRP" ] ) )
-hierarchy = scriptWindow.getLayout().editors( GafferSceneUI.HierarchyView )[0]
 __delay( 0.1 )
-GafferUI.WidgetAlgo.grab( widget = hierarchyView, imagePath = "images/hierarchySceneExpandedTwoLevels.png" )
+GafferUI.WidgetAlgo.grab( widget = hierarchyView, imagePath = "images/hierarchyViewExpandedTwoLevels.png" )
```

Comment:
We need to update index.md to refer to this new image.

Reply:
Can't believe I missed that. Fixed.